bio/minibwa/index/wrapper.py

```
@@ -0,0 +1,31 @@
+from tempfile import TemporaryDirectory
+from snakemake.shell import shell
+from snakemake_wrapper_utils.snakemake import move_files, is_arg
+
+extra = snakemake.params.get("extra", "")
+log = snakemake.log_fmt_shell(stdout=True, stderr=True, append=True)
+
+# There is no way to control output file name, which is why we have to move them.
+with TemporaryDirectory() as tempdir:
+    genome_link = f"{tempdir}/genome.fasta"
+    from pathlib import Path
+    Path(genome_link).symlink_to(snakemake.input[0]) 
```

Ruff (0.16.2)

[error] Undefined name `snakemake`

(F821)